crates/iota-types/Cargo.toml

```diff
@@ -19,8 +19,7 @@ better_any.workspace = true
 bincode.workspace = true
 byteorder.workspace = true
 chrono.workspace = true
-derivative.workspace = true
-derive_more = "0.99.17"
+derive_more = { workspace = true, features = ["as_ref", "from", "debug", "display"] }
```

Comment:
Are we using `as_ref` and `from`?

Reply:
Yes, in `iota-types crypto.rs`

Reply:
How come the features weren't there then?

Reply:
They used to be default, now they are not.

Reply:
I see, thanks